src/man/firejail.txt

```diff
@@ -1144,6 +1144,33 @@ Example:
 .br
 $ firejail --keep-var-tmp
 
+#ifdef HAVE_LANDLOCK
```

Comment:
I think there should be a LANDLOCK section explain the concept and the limitations of landlock.